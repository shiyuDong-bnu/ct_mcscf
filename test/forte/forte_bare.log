2025-02-18:15:22:17,644 INFO     [psi4.driver.task_planner:285] PLANNING Atomic:  keywords={'D_CONVERGENCE': 1e-08, 'E_CONVERGENCE': 1e-10, 'MAXITER': 300, 'REFERENCE': 'RHF', 'SCF_TYPE': 'PK', 'forte__job_type': 'MCSCF_TWO_STEP', 'forte__scf_type': 'PK', 'forte__e_convergence': 1e-08, 'forte__d_convergence': 1e-08, 'forte__active_space_solver': 'FCI', 'forte__active': [2], 'forte__r_convergence': 1e-08, 'forte__casscf_micro_maxiter': 4, 'forte__casscf_g_convergence': 1e-06, 'forte__reference': 'RHF', 'forte__maxiter': 300, 'forte__basis': 'CC-PVDZ', 'function_kwargs': {}}
2025-02-18:15:22:17,646 INFO     [psi4.driver.driver:523] Compute energy(): method=forte, basis=cc-pvdz, molecule=CO, nre=0.52917721067
2025-02-18:15:22:17,785 INFO     [psi4.driver.driver:526] Return energy(): -1.12719939987895
