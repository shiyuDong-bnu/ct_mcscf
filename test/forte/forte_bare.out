
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.9.1 release

                         Git: Rev {} zzzzzzz 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, M. H. Lechner, A. Jiang, S. Behnle, A. G. Heide,
    M. F. Herbst, and D. L. Poole

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Tuesday, 18 February 2025 03:22PM

    Process ID: 2693914
    Host:       debianlab
    PSIDATADIR: /home/sydong/miniconda3/envs/p4env/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
import forte

molecule CO{
  0 1
  H  
  H  1 1.0
  symmetry c1
}

set {
  basis                 cc-pvdz
  reference             rhf
  scf_type              pk
  maxiter               300
  e_convergence         10
  d_convergence         8
#  docc                  [5,0,1,1]
}

set forte {
  job_type              mcscf_two_step
  active_space_solver                fci
#  restricted_docc       [2]
  active                [2,]
#  int_type              fcidump
#  fcidump_file          BARE.DUMP 
  e_convergence         8  # energy convergence of the FCI iterations
  r_convergence         8  # residual convergence of the FCI iterations
  casscf_e_convergence  8  # energy convergence of the MCSCF iterations
  casscf_g_convergence  6  # gradient convergence of the MCSCF iterations
  casscf_micro_maxiter  4  # do at least 4 micro iterations per macro iteration
}

Eforte = energy('forte')
--------------------------------------------------------------------------

Scratch directory: /tmp/

  Forte
  ----------------------------------------------------------------------------
  A suite of quantum chemistry methods for strongly correlated electrons

    git branch:  - git commit: 

  Developed by:
    Francesco A. Evangelista, Chenyang Li, Kevin P. Hannon,
    Jeffrey B. Schriber, Tianyuan Zhang, Chenxi Cai,
    Nan He, Nicholas Stair, Shuhe Wang, Renke Huang
  ----------------------------------------------------------------------------


  Preparing forte objects from a Psi4 Wavefunction object
  No reference wave function provided for Forte. Computing SCF orbitals using Psi4 ...
   => Libint2 <=

    Primary   basis highest AM E, G, H:  6, 6, 3
    Auxiliary basis highest AM E, G, H:  7, 7, 4
    Onebody   basis highest AM E, G, H:  -, -, -
    Solid Harmonics ordering:            Gaussian

*** tstart() called on debianlab
*** at Tue Feb 18 15:22:17 2025

   => Loading Basis Set <=

    Name: CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1-2 entry H          line    22 file /home/sydong/miniconda3/envs/p4env/share/psi4/basis/cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: D_inf_h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         H            0.000000000000     0.000000000000    -0.500000000000     1.007825032230
         H            0.000000000000     0.000000000000     0.500000000000     1.007825032230

  Running in c1 symmetry.

  Rotational constants: A = ************  B =     33.45348  C =     33.45348 [cm^-1]
  Rotational constants: A = ************  B = 1002910.21295  C = 1002910.21295 [MHz]
  Nuclear repulsion =    0.529177210670000

  Charge       = 0
  Multiplicity = 1
  Electrons    = 2
  Nalpha       = 1
  Nbeta        = 1

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-10
  Density threshold  = 1.00e-08
  Integral threshold = 1.00e-12

  ==> Primary Basis <==

  Basis Set: CC-PVDZ
    Blend: CC-PVDZ
    Number of shells: 6
    Number of basis functions: 10
    Number of Cartesian functions: 10
    Spherical Harmonics?: true
    Max angular momentum: 1

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   2
      Number of AO shells:               6
      Number of primitives:             10
      Number of atomic orbitals:        10
      Number of basis functions:        10

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 3080 doubles for integral storage.
  We computed 231 shell quartets total.
  Whereas there are 231 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 1.1235676082E-01.
  Reciprocal condition number of the overlap matrix is 3.8775501553E-02.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     A         10      10 
   -------------------------
    Total      10      10
   -------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @RHF iter SAD:    -0.83821031082802   -8.38210e-01   0.00000e+00 
   @RHF iter   1:    -1.09846572476544   -2.60255e-01   3.87301e-03 ADIIS/DIIS
   @RHF iter   2:    -1.10010178613905   -1.63606e-03   6.74342e-04 ADIIS/DIIS
   @RHF iter   3:    -1.10015371126237   -5.19251e-05   2.87927e-05 DIIS
   @RHF iter   4:    -1.10015376478324   -5.35209e-08   3.00125e-07 DIIS
   @RHF iter   5:    -1.10015376479352   -1.02882e-11   1.91210e-10 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A     -0.525044  

    Virtual:                                                              

       2A      0.143588     3A      0.539607     4A      0.794799  
       5A      1.287130     6A      1.287130     7A      1.679162  
       8A      1.871551     9A      1.871551    10A      2.735411  

    Final Occupation by Irrep:
              A 
    DOCC [     1 ]
    NA   [     1 ]
    NB   [     1 ]

  @RHF Final Energy:    -1.10015376479352

   => Energetics <=

    Nuclear Repulsion Energy =              0.5291772106700000
    One-Electron Energy =                  -2.2085745472379776
    Two-Electron Energy =                   0.5792435717744545
    Total Energy =                         -1.1001537647935233

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix


 Multipole Moments:

 ------------------------------------------------------------------------------------
     Multipole            Electronic (a.u.)      Nuclear  (a.u.)        Total (a.u.)
 ------------------------------------------------------------------------------------

 L = 1.  Multiply by 2.5417464519 to convert [e a0] to [Debye]
 Dipole X            :         -0.0000000            0.0000000           -0.0000000
 Dipole Y            :         -0.0000000            0.0000000           -0.0000000
 Dipole Z            :         -0.0000000            0.0000000           -0.0000000
 Magnitude           :                                                    0.0000000

 ------------------------------------------------------------------------------------

*** tstop() called on debianlab at Tue Feb 18 15:22:17 2025
Module time:
	user time   =       0.13 seconds =       0.00 minutes
	system time =       0.00 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.13 seconds =       0.00 minutes
	system time =       0.00 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes

  Read options for space ACTIVE

  ==> MO Space Information <==

  -------------------------------
                        A   Sum
  -------------------------------
    FROZEN_DOCC         0     0
    RESTRICTED_DOCC     0     0
    GAS1                2     2
    GAS2                0     0
    GAS3                0     0
    GAS4                0     0
    GAS5                0     0
    GAS6                0     0
    RESTRICTED_UOCC     8     8
    FROZEN_UOCC         0     0
    Total              10    10
  -------------------------------   => Loading Basis Set <=

    Name: STO-3G
    Role: ORBITAL
    Keyword: MINAO_BASIS
    atoms 1-2 entry H          line    19 file /home/sydong/miniconda3/envs/p4env/share/psi4/basis/sto-3g.gbs 


  State Singlet (Ms = 0) A GAS min: 0 0 0 0 0 0 ; GAS max: 4 0 0 0 0 0 ; weights:
      1.000000000000
  Forte will use psi4 integrals

  ==> Primary Basis Set Summary <==

  Basis Set: CC-PVDZ
    Blend: CC-PVDZ
    Number of shells: 6
    Number of basis functions: 10
    Number of Cartesian functions: 10
    Spherical Harmonics?: true
    Max angular momentum: 1


  JK created using conventional PK integrals
  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   2
      Number of AO shells:               6
      Number of primitives:             10
      Number of atomic orbitals:        10
      Number of basis functions:        10

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 3080 doubles for integral storage.
  We computed 231 shell quartets total.
  Whereas there are 231 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              400
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1



  ==> Integral Transformation <==

  Number of molecular orbitals:                         10
  Number of correlated molecular orbitals:              10
  Number of frozen occupied orbitals:                    0
  Number of frozen unoccupied orbitals:                  0
  Two-electron integral type:                 Conventional


  Skip integral allocation and transformation for AO-driven CASSCF.

          -----------------------------------------------------------
                  Multi-Configurational Self Consistent Field
                Two-Step Approximate Second-Order AO Algorithm
            written by Chenyang Li, Kevin P. Hannon, and Shuhe Wang
          -----------------------------------------------------------


  ==> MCSCF Calculation Information <==

    --------------------------------------------------------
    Print level                                      Default
    Integral type                               CONVENTIONAL
    CI solver type                                       FCI
    Final orbital type                             CANONICAL
    Derivative type                                     NONE
    Optimize orbitals                                   TRUE
    Include internal rotations                         FALSE
    Debug printing                                     FALSE
    Energy convergence                             1.000e-08
    Gradient convergence                           1.000e-06
    Max value for rotation                         2.000e-01
    Max number of macro iterations                       100
    Max number of micro iterations                         4
    Min number of micro iterations                         4
    DIIS start                                            15
    Min DIIS vectors                                       3
    Max DIIS vectors                                       8
    Frequency of DIIS extrapolation                        1
    --------------------------------------------------------

  ==> Independent Orbital Rotations <==

    ORBITAL SPACES                         A
    ----------------------------------------
             ACTIVE / RESTRICTED_DOCC      0
    RESTRICTED_UOCC /          ACTIVE     16
    RESTRICTED_UOCC / RESTRICTED_DOCC      0
    ----------------------------------------

  ==> MCSCF Iterations <==

                      Energy CI                    Energy Orbital
           ------------------------------  ------------------------------
    Iter.        Total Energy       Delta        Total Energy       Delta  Orb. Grad.  Micro
    ----------------------------------------------------------------------------------------
       2      -1.108337719536 -1.1083e+00     -1.123273900727 -1.1233e+00  9.0243e-03    4/N
       3      -1.126563458420 -1.8226e-02     -1.127085066830 -3.8112e-03  2.9149e-03    4/N
       4      -1.127184809739 -6.2135e-04     -1.127197481876 -1.1242e-04  7.3291e-05    4/N
       5      -1.127199162568 -1.4353e-05     -1.127199370329 -1.8885e-06  1.4192e-05    4/N
       6      -1.127199396239 -2.3367e-07     -1.127199399430 -2.9101e-08  1.8727e-06    4/N
       7      -1.127199399824 -3.5854e-09     -1.127199399873 -4.4327e-10  1.4902e-07    2/Y
    ----------------------------------------------------------------------------------------

  A miracle has come to pass: MCSCF iterations have converged!

  Performing final CI Calculation using converged orbitals

  ==> String Lists <==

    --------------------------------------------------------
    number of alpha electrons                              1
    number of beta electrons                               1
    number of alpha strings                                2
    number of beta strings                                 2
    --------------------------------------------------------

  ==> FCI Solver <==

    --------------------------------------------------------
    Spin adapt                                         FALSE
    Number of determinants                                 4
    Symmetry                                               0
    Multiplicity                                           1
    Number of roots                                        1
    Target root                                            0
    --------------------------------------------------------

  ==> Initial Guess <==

  Initial guess determinants:         4

  Classification of the initial guess solutions

  Number   2S+1   Selected
  ------------------------
     3       1       *
     1       3        
  ------------------------

    Spin    Root           Energy        <S^2>    Status
  -------------------------------------------------------
   singlet    0       -1.127199399879  +0.000000  added
  -------------------------------------------------------

  ==> Root No. 0 <==

    20      0.98685736
    02     -0.16159378

    Total Energy:      -1.127199399879, <S^2>: -0.000000
    Time for FCI:       0.000071008000

  ==> Energy Summary <==

    Multi.(2ms)  Irrep.  No.               Energy      <S^2>
    --------------------------------------------------------
       1  (  0)     A     0       -1.127199399879  -0.000000
    --------------------------------------------------------

  ==> Natural Orbitals <==

        1A      1.947775      2A      0.052225  


  ==> Dipole Moments [e a0] (Nuclear + Electronic) for Singlet (Ms = 0) A <==

       State           DM_X           DM_Y           DM_Z           |DM|
    --------------------------------------------------------------------
          0A    -0.00000000    -0.00000000    -0.00000000     0.00000000
    --------------------------------------------------------------------
     Nuclear     0.00000000     0.00000000     0.00000000     0.00000000
    --------------------------------------------------------------------

  ==> Natural Orbitals <==

        1A      1.947775      2A      0.052225  


  ==> Quadrupole Moments [e a0^2] (Nuclear + Electronic) for Singlet (Ms = 0) A <==

       State          QM_XX          QM_XY          QM_XZ          QM_YY          QM_YZ          QM_ZZ
    --------------------------------------------------------------------------------------------------
          0A    -1.84157501    -0.00000000    -0.00000000    -1.84157501     0.00000000    -1.33279804
    --------------------------------------------------------------------------------------------------
     Nuclear     0.00000000     0.00000000     0.00000000     0.00000000     0.00000000     1.78553241
    --------------------------------------------------------------------------------------------------

  ==> Natural Orbitals <==

        1A      1.947775      2A      0.052225  


  ==> Semicanonicalize Orbitals <==

    MIX INACTIVE ORBITALS         FALSE
    MIX GAS ACTIVE ORBITALS       FALSE
    FROZEN_DOCC                   CANONICAL
    FROZEN_UOCC                   CANONICAL
    GAS1                          CANONICAL
    RESTRICTED_DOCC               CANONICAL
    RESTRICTED_UOCC               CANONICAL

    Off-Diag. Elements       Max           2-Norm
    ------------------------------------------------
    GAS1                 0.0000000000   0.0000000000
    RESTRICTED_UOCC      0.0775104957   0.1096390302
    ------------------------------------------------

    Canonicalization test failed

  Timing for orbital canonicalization:                        0.000 s.

  Time to prepare integrals:        0.131 seconds
  Time to run job          :        0.007 seconds
  Total                    :        0.138 seconds

    Psi4 stopped on: Tuesday, 18 February 2025 03:22PM
    Psi4 wall time for execution: 0:00:00.20

*** Psi4 exiting successfully. Buy a developer a beer!
